src/dpp/cluster/thread.cpp

```diff
@@ -70,15 +70,41 @@ void cluster::threads_get_private_archived(snowflake channel_id, time_t before_t
 		{"before", before_timestamp},
 		{"limit", limit},
 	});
-	rest_request_list<thread>(this, API_PATH "/channels", std::to_string(channel_id), "/threads/archived/private" + parameters, m_get, "", callback);
+	this->post_rest(API_PATH "/channels", std::to_string(channel_id), "/threads/archived/private" + parameters, m_get, "", [this, callback](json &j, const http_request_completion_t& http) {
+		std::unordered_map<snowflake, thread> list;
+		confirmation_callback_t e(this, confirmation(), http);
```

Comment:
What isn't used in that?

Reply:
No, you seem to create a new confirmation_callback_t object below to use with the callback

Reply:
It is possible that I am missing something though

Reply:
This is the same as in cluster::threads_get_active from line 34 in the same file

Reply:
This is necessary for checking rest exception errors. The supplied callback is not a confirmation_callback_t, and another confirmation_callback_t must be created with the list instead of the confirmation() below. Nothing here is unused.

Reply:
if you dont use rest_request_list it won't properly generate coro/sync versions